src/SprykerEco/Zed/Payone/Business/Distributor/OrderPriceDistributor.php

```diff
@@ -70,13 +75,13 @@ protected function findPayonePayment(OrderTransfer $orderTransfer): ?PaymentTran
     }
 
     /**
-     * @param int $unitPrice
+     * @param int|null $unitPrice
      * @param float $priceRatio
      * @param float $roundingError
      *
      * @return int
      */
-    protected function calculateRoundedPrice(int $unitPrice, float $priceRatio, float &$roundingError): int
+    protected function calculateRoundedPrice(?int $unitPrice, float $priceRatio, float &$roundingError): int
```

Comment:
either wrong or should be passed with null default value

Reply:
looks as wrong attempt to calculate null price

Reply:
Fixed.